packages/next/src/elements/Nav/index.client.tsx

```diff
@@ -44,7 +44,10 @@ export const DefaultNavClient: React.FC<{
                 id = `nav-global-${slug}`
               }
 
-              const isActive = pathname.startsWith(href)
+              const isActive =
```

Comment:
We should go back to the logic previous to #11457 here: https://github.com/payloadcms/payload/pull/11457/files#diff-a17efb725de24019c26e12278e1f4cd475664d3a9a5b2152bac1728282ad2695L47